src/www/ui/ajax-notice-files.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/***********************************************************
+ Copyright (C) 2019 Siemens AG
+ Author: Andreas J. Reichel <andreas.reichel@tngtech.com>
+
+ This program is free software; you can redistribute it and/or
+ modify it under the terms of the GNU General Public License
+ version 2 as published by the Free Software Foundation.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License along
+ with this program; if not, write to the Free Software Foundation, Inc.,
+ 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ ***********************************************************/
+
+include_once "search-helper.php";
+include_once "common-repo.php";
```

Comment:
I am getting error because of this:
```
PHP Warning:  include_once(common-repo.php): failed to open stream: No such file or directory in /usr/local/share/fossology/www/ui/ajax-notice-files.php on line 21
PHP Warning:  include_once(): Failed opening 'common-repo.php' for inclusion (include_path='.:/usr/share/php') in /usr/local/share/fossology/www/ui/ajax-notice-files.php on line 21
```

Following changes fix that.

```suggestion
require_once dirname(dirname(dirname(__FILE__))) . "/lib/php/common-repo.php";
```